setup.py

```diff
@@ -34,7 +34,8 @@
     'package_dir': {'fontFeatures': 'fontFeatures'},
     'packages': find_packages(exclude=["*.tests", "*.tests.*", "tests.*", "tests"])
 ,
-    'scripts': scripts
+    'scripts': scripts,
+    'zip_safe': False
```

Comment:
Why? (You're probably right, I just don't understand this.)

Reply:
It allows me to edit the installed package if needed. Obviously you normally shouldn't do that but it is convenient for quick debugging.